application/testing/CMakeLists.txt

```diff
@@ -94,9 +94,8 @@ function(f3d_test)
     set_tests_properties(f3d::${F3D_TEST_NAME} PROPERTIES FIXTURES_REQUIRED f3d::${F3D_TEST_DEPENDS}_FIXTURE)
   endif()
 
-  if(F3D_TEST_FORCE_PROGRESS_BAR)
-    set_tests_properties(f3d::${F3D_TEST_NAME} PROPERTIES ENVIRONMENT "CTEST_PROGRESS_BAR_TESTING=1")
-  endif()
+  set_tests_properties(f3d::${F3D_TEST_NAME} PROPERTIES ENVIRONMENT
```

Comment:
@mwestphal I've removed `FORCE_PROGRESS_BAR` and set the variable all the time for the tests. I don't think it hurts

Reply:
fair enough